collection_modules/btle_beacon/collectionPoint.py

```diff
@@ -1,80 +1,89 @@
 """
-BTLE iBeacon collection module
-WIP 
-
-RegisteredClientRegistery manages the list of clients that are in rage
-event manager controllers and handles events and figures out if the event needs to be handled and put in the list of registered clients
+BTLE iBeacon module
+Main collection point
 """
```

Comment:
you whine about my no in line comments and take out my class level comments.  that seems like a funky move

Reply:
I moved these comments to their classes, where they belong.